### PR TITLE
Upgrade to v3.64.2 of the AWS Terraform Provider

### PR DESCRIPTION
**Pull request by stack72**
Fixes: #1536 
Fixes: #1687 
Fixes: #1688
Fixes: #1689

The breaking changes are not strictly breaking - we are removing these as inputs as they should only be outputs

### CODE REVIEW

File: sdk/dotnet/AppStream/Fleet.cs [hunk 1]
```diff
@@ -335,14 +335,6 @@ public InputMap<string> Tags
             set => _tags = value;
         }
 
-        [Input("tagsAll")]
-        private InputMap<string>? _tagsAll;
-        public InputMap<string> TagsAll
```
Comment (jkodroff) [inline sdk/dotnet/AppStream/Fleet.cs:340]: Breaking change?  I don't see anything in the upstream CHANGELOG.

File: sdk/dotnet/Connect/ContactFlow.cs [hunk 2]
```diff
@@ -240,18 +240,6 @@ public InputMap<string> Tags
             set => _tags = value;
         }
 
-        [Input("tagsAll")]
-        private InputMap<string>? _tagsAll;
-
-        /// <summary>
-        /// A map of tags assigned to the resource, including those inherited from the provider [`default_tags` configuration block](https://www.terraform.io/docs/providers/aws/index.html#default_tags-configuration-block).
-        /// </summary>
-        public InputMap<string> TagsAll
```
Comment (jkodroff) [inline sdk/dotnet/Connect/ContactFlow.cs:249]: Breaking change?

File: sdk/dotnet/DirectConnect/Connection.cs [hunk 3]
```diff
@@ -191,18 +191,6 @@ public InputMap<string> Tags
             set => _tags = value;
         }
 
-        [Input("tagsAll")]
-        private InputMap<string>? _tagsAll;
-
-        /// <summary>
-        /// A map of tags assigned to the resource, including those inherited from the provider `default_tags` configuration block.
-        /// </summary>
-        public InputMap<string> TagsAll
```
Comment (jkodroff) [inline sdk/dotnet/DirectConnect/Connection.cs:200]: Potential breaking change.

File: sdk/dotnet/Ec2/DedicatedHost.cs [hunk 4]
```diff
@@ -179,14 +182,6 @@ public InputMap<string> Tags
             set => _tags = value;
         }
 
-        [Input("tagsAll")]
-        private InputMap<string>? _tagsAll;
-        public InputMap<string> TagsAll
```
Comment (jkodroff) [inline sdk/dotnet/Ec2/DedicatedHost.cs:184]: Breaking change?

File: sdk/dotnet/Ec2/PlacementGroup.cs [hunk 5]
```diff
@@ -61,7 +61,7 @@ public partial class PlacementGroup : Pulumi.CustomResource
         /// `"partition"`.  Valid values are 1 - 7 (default is `2`).
         /// </summary>
         [Output("partitionCount")]
-        public Output<int?> PartitionCount { get; private set; } = null!;
```
Comment (jkodroff) [inline sdk/dotnet/Ec2/PlacementGroup.cs:64]: Breaking change, but less than 1% of users.

File: sdk/dotnet/Fsx/Backup.cs [hunk 6]
```diff
@@ -158,18 +158,6 @@ public InputMap<string> Tags
             set => _tags = value;
         }
 
-        [Input("tagsAll")]
-        private InputMap<string>? _tagsAll;
-
-        /// <summary>
-        /// A map of tags assigned to the resource, including those inherited from the provider [`default_tags` configuration block](https://www.terraform.io/docs/providers/aws/index.html#default_tags-configuration-block).
-        /// </summary>
-        public InputMap<string> TagsAll
```
Comment (jkodroff) [inline sdk/dotnet/Fsx/Backup.cs:167]: Breaking change?

**Review by jkodroff — APPROVED** [review]
Not really feasible to be an exhaustive search due to the large number of changed files.  What I can see is:

- `TagsAll` is removed as an input in a lot of places.
- Most of the other changes are around docs, many of which add a comma after "e.g." in the docs.

If you're ok with the `TagsAll` changes, should be ok.

File: provider/resources.go [hunk 7]
```diff
@@ -289,6 +290,7 @@ func Provider() tfbridge.ProviderInfo {
 		Homepage:    "https://pulumi.io",
 		Repository:  "https://github.com/pulumi/pulumi-aws",
 		Version:     version.Version,
+		GitHubOrg:   "hashicorp",
```
[the remark this answers is not in the document]
Comment (stack72) [reply provider/resources.go:293]: We had a fork so it was picking up the fork :)

**Pull request merged**